### PR TITLE
Clean up intro

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,30 +21,94 @@ For macOS we provide installers [on GitHub](https://github.com/CanonicalLtd/mult
...
 
-## Launch an instance
+## Find available images
+```
+$ multipass find
+Image  Aliases         Version   Description
+core                   20180419  Ubuntu Core 16
+14.04  t, trusty       20181203  Ubuntu 14.04 LTS
+16.04  x, xenial       20181207  Ubuntu 16.04 LTS
+18.04  b, bionic, lts  20181207  Ubuntu 16.04 LTS
```
Comment (debsahu) [inline README.md]: `18.04  b, bionic, lts  20181207  Ubuntu 16.04 LTS` should be changed to `18.04  b, bionic, lts  20181207  Ubuntu 18.04 LTS`

File: README.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
 # What is multipass?
 
-It's a system that orchestrates the creation, management and maintenance
-of virtual machines and associated Ubuntu images to simplify development.
+Multipass is a lightweight VM manager for Linux and MacOS. It's designed for
+developers who want a fresh Ubuntu with a single command or hotkey. It uses
```
Comment (townsend2010) [inline README.md]: I would say "a fresh Ubuntu virtual machine" here. Saying "a fresh Ubuntu with" seems like Ubuntu itself is the noun, but really should be the adjective in this case.

Also, we don't support a hotkey yet, so maybe we should leave that out for now?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,17 @@
 # What is multipass?
 
-It's a system that orchestrates the creation, management and maintenance
-of virtual machines and associated Ubuntu images to simplify development.
+Multipass is a lightweight VM manager for Linux and MacOS. It's designed for
```
Comment (townsend2010) [inline README.md]: We can mention Windows now too:)

File: README.md [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
+developers who want a fresh Ubuntu with a single command or hotkey. It uses
+KVM on Linux and HyperKit on MacOS to run the VM with minimal overhead.
```
Comment (townsend2010) [inline README.md]: Mention Hyper-V as well.

File: README.md [hunk 5]
```diff
@@ -1,11 +1,17 @@
...
+Multipass will fetch images for you and keep them up to date.
 
-# Getting it
+Since it supports metadata for cloud-init, you can simulate a small cloud
+deployment on your laptop or workstation. See [Using cloud-init with Multipass](https://blog.ubuntu.com/2018/04/02/using-cloud-init-with-multipass)
+on the Ubuntu blog for more details.
 
-On Linux it's available as a classically confined snap, in the `beta` channel:
+# Install Multipass
```
Comment (townsend2010) [inline README.md]: Should also mention the Windows way to install.

**Comment by Saviq** [comment]
I've updated this now. Sorry it took so long…

File: README.md [hunk 6]
```diff
@@ -1,11 +1,17 @@
 # What is multipass?
 
-It's a system that orchestrates the creation, management and maintenance
-of virtual machines and associated Ubuntu images to simplify development.
+Multipass is a lightweight VM manager for Linux, Windows and macOS. It's designed
```
Comment (townsend2010) [inline README.md]: Capitalize the 'm' for 'MacOS'.

File: README.md [hunk 7]
```diff
@@ -1,11 +1,17 @@
...
+for developers who want a fresh Ubuntu environment with a single command. It uses
+KVM on Linux, Hyper-V on Windows HyperKit on MacOS to run the VM with minimal
```
Comment (townsend2010) [inline README.md]: Need `, and` between `Windows` and `HyperKit`.

File: README.md (hunk 6 above)
Comment (townsend2010) [inline README.md]: Capitalize the 'm' for 'MacOS'.

File: README.md [hunk 8]
```diff
@@ -1,11 +1,17 @@
...
+overhead. It can also use VirtualBox on Windows and macOS.
```
Comment (townsend2010) [inline README.md]: Capital m here for MacOS.

File: README.md (hunk 6 above)
Comment (Saviq) [reply README.md]: [No](https://www.apple.com/macos/).

File: README.md (hunk 7 above)
Comment (Saviq) [reply README.md]: Done.

File: README.md (hunk 8 above)
Comment (Saviq) [reply README.md]: ditto

File: README.md (hunk 6 above)
Comment (townsend2010) [reply README.md]: Ok, I stand corrected :smiley: I was basing that on the place where it _was_ `MacOS` (that you've since fixed).

**Pull request merged**